src/lib/root_prover/root_prover.ml

```diff
@@ -107,7 +107,7 @@ module Make (Inputs : Inputs_intf) :
       ~metadata:
         [ ( "merkle_list"
           , `List (List.map ~f:State_body_hash.to_yojson merkle_list) ) ]
-      "Produced a merkle list of $merkle_list" ;
+      "Root prover produced a merkle list of $merkle_list" ;
```

Comment:
discussed with @bkase this may be too low level - let's move this to debug. Rationale is that this is very in the weeds of the bootstrapping process.

Also, is this dead code? 

Could someone else confirm both these assumptions? cc @nholland94